internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -625,6 +625,7 @@ func WriteUdevRules(runtime.Sequence, any) (runtime.TaskExecutionFunc, string) {
 			content.WriteByte('\n')
 		}
 
+		// TODO: SELinux label?
```

Comment:
We should perhaps label our custom rules as well. I thought labels for rules will come later, but will do now